paddle/fluid/operators/jit/gen_base.cc

```diff
@@ -40,6 +46,17 @@ void GenBase::dumpCode(const unsigned char* code) const {
   }
 }
 
+void* GenBase::operator new(size_t size) {
```

Comment:
Yes, but once again, are these needed?
As you said, no such AVX instructions are used with these classes, thus it would make sense to just remove the alignment requirement.